infra/modules/ai/cognitiveservices.bicep

```
@@ -92,7 +95,7 @@ resource deploymentScript 'Microsoft.Resources/deploymentScripts@2023-08-01' = [
     }
   }
   properties: {
-    azCliVersion: '2.9.1'
+    azCliVersion: '2.47.0'
     scriptContent: 'az rest --method put --url ${environment().resourceManager}/subscriptions/${subscription().subscriptionId}/resourceGroups/${resourceGroup().name}/providers/Microsoft.CognitiveServices/accounts/${account.name}/raiPolicies/${raiPolicy.name}?api-version=2023-10-01-preview --debug --body "${raiPolicy.payload}"'
     cleanupPreference: 'OnExpiration'
     retentionInterval: 'PT1H'
```

```suggestion
    retentionInterval: 'PT1H'
    forceUpdateTag: guid(raiPolicy.payload)
```

It seems you didn't include the forceUpdateTag from https://github.com/Azure/enterprise-azureai/pull/37/files. This is required to update the content filter when the file changes.

---

missed that, it's also not in the current main, but added